flask_hw5/blueprints/rooms/rooms_routes.py

```diff
@@ -0,0 +1,46 @@
+from flask_restful import Resource
+from blueprints.rooms.room_arg_parser import (room_parser,
+                                              room_parser_post,
+                                              room_parser_patch)
+from blueprints.rooms.room_object_serlzer import marshal_with_decor
+from blueprints.rooms.rooms_model import Room
+from hotel_db import db
+
+
+class Rooms(Resource):
+    @marshal_with_decor
+    def get(self):
+        room_filters = room_parser.parse_args()
+        if any(tuple(room_filters.values())):
+            return Room.query.filter_by(
+                                        **{k: v
+                                           for k, v in room_filters.items()
+                                           if v}
+                                       ).first()
+        return Room.query.all(), 200
+
+    def post(self):
+        new_room_values = room_parser_post.parse_args()
+        db.session.add(Room(**new_room_values))
+        db.session.commit()
+        return 'New room was successfully added!', 200
```

Comment:
return added room. Client should know what fields you added

Reply:
Done